integration_tests/dbt_project.yml

```diff
@@ -12,6 +12,7 @@ dispatch:
 
 vars:
   pendo_source:
+    pendo_schema: pendo_integration_tests_2
```

Comment:
This should be consistent with the schema used in all CircleCi warehouse configs